docs/index.rst

```diff
@@ -84,17 +84,17 @@ User Documentation
    io/ascii/index
    io/votable/index
    io/misc
+   vo/index
```

Comment:
I guess my point is that if one had tools to download data from the HST or ESO archive, they would be under I/O as well (actually, maybe `vo` can already do that??).